lib/jsdom/browser/Window.js

```diff
@@ -146,6 +149,24 @@ function Window(options) {
 
   this.postMessage = postMessage;
 
+  this.atob = function (str) {
+    const result = atob(str);
+    if (result === null) {
+      throw new DOMException(DOMException.INVALID_CHARACTER_ERR,
+        "The string to be encoded contains characters outside of the Latin1 range.");
+    }
+    return result;
+  };
+
+  this.btoa = function (str) {
+    const result = btoa(str);
+    if (result === null) {
+      throw new DOMException(DOMException.INVALID_CHARACTER_ERR,
+        "The string to be encoded contains characters outside of the Latin1 range.");
```

Comment:
I don't think this is quite the right error message, at least for btoa. Firefox shows "String contains an invalid character".